packages/webpack-cli/README.md

```diff
@@ -1,4 +1,15 @@
-# `webpack-cli`
+<div align="center">
+    <a href="https://github.com/webpack/webpack-cli">
+        <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
+    </a>
+</div>
+
+<h1 align="center">webpack CLI</h1>
+
+<p align="center">
+  The official CLI of webpack
+</p>
+<br>
```

Comment:
Let's prefer to use markdown in markdown, like:
```md
<div align="center">
    <a href="https://github.com/webpack/webpack-cli">
        <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
    </a>
</div>

# webpack CLI

The official CLI of webpack
```

Reply:
done

Reply:
I'll suggest keeping HTML to center the headings and description, can't center with MD

Reply:
It is documentation, not web page, so you read this from left to right (in most of languages), no need centered it (especial on very large monitors )

Reply:
NPM doesn't center the headings anyway (https://www.npmjs.com/package/webpack) so no need.
Also, dev server does same as well (https://github.com/webpack/webpack-dev-server/blob/master/README.md).

Reply:
seems fair